src/test/resources/application.properties

```diff
@@ -11,4 +11,10 @@ db.stairway.username=stairwayuser
 db.stairway.password=stairwaypwd
 db.stairway.migrateUpgrade=true
 db.stairway.forceClean=true
-samService.basePath=https://sam.dsde-dev.broadinstitute.org
\ No newline at end of file
+samService.basePath=https://sam.dsde-dev.broadinstitute.org
+
+it.vaultAddress=https://clotho.broadinstitute.org:8200
+it.vaultPath=secret/dsde/firecloud/alpha/common/firecloud-account.json
+it.createWorkspaceUrlDev=http://34.71.179.202/dev-terra-workspace-manager/api/v1/workspaces
+it.serviceAccountEmail=william.thunderlord@test.firecloud.org
+it.vaultTokenFileName=.vault-token
```

Comment:
see my previous comment; this should be templatized and rendered before tests run - which would help with the ability to target different environments. If we wanted to switch from testing against dev to testing against alpha (for instance), we'd just need to re-render configs before running the tests.

Reply:
Very well noted! Thanks for all the feedback, by the way!

Reply:
Done!